arangod/Aql/MultiGet.h

```diff
@@ -58,15 +58,14 @@ class MultiGetContext {
 #ifdef ARANGODB_USE_GOOGLE_TESTS
     auto* physical = dynamic_cast<RocksDBMetaCollection*>(base);
     if (ADB_UNLIKELY(physical == nullptr)) {
-      base->lookup(&trx, id,
-                   IndexIterator::makeDocumentCallbackF(
-                       [&](LocalDocumentId, VPackSlice slice) {
-                         values[_global] =
-                             std::make_unique<std::string>(std::string_view{
-                                 slice.startAs<char>(), slice.byteSize()});
-                         return true;
-                       }),
-                   {.readCache = false, .fillCache = false}, snapshot);
+      base->lookup(
+          &trx, id,
+          [&](LocalDocumentId, ValueType&&, VPackSlice doc) {
+            values[_global] = std::make_unique<std::string>(
+                std::string_view{doc.startAs<char>(), doc.byteSize()});
```

Comment:
Is it possible that `ValueType` is set here? We could then even use the unique_ptr from ValueType without creating yet another string.

Reply:
Yes, but it's code only for mock collection, so I don't see the profit for optimizing it